code/modules/economy/trader.dm

```diff
@@ -1410,31 +1406,370 @@
+//        ██████  ██░ ██  ▄▄▄      ▓█████▄ ▓██   ██▓    ▄▄▄██▀▀▀ ▄▄▄       ███▄    █ ▓█████     //
+//      ▒██    ▒ ▓██░ ██▒▒████▄    ▒██▀ ██▌ ▒██  ██▒      ▒██   ▒████▄     ██ ▀█   █ ▓█   ▀     //
+//      ░ ▓██▄   ▒██▀▀██░▒██  ▀█▄  ░██   █▌  ▒██ ██░      ░██   ▒██  ▀█▄  ▓██  ▀█ ██▒▒███       //
+//       ▒   ██▒░▓█ ░██ ░██▄▄▄▄██ ░▓█▄   ▌  ░ ▐██▓░   ▓██▄██▓  ░██▄▄▄▄██ ▓██▒  ▐▌██▒▒▓█  ▄      //
+//     ▒██████▒▒░▓█▒░██▓ ▓█   ▓██▒░▒████▓   ░ ██▒▓░    ▓███▒    ▓█   ▓██▒▒██░   ▓██░░▒████▒     //
+//      ▒ ▒▓▒ ▒ ░ ▒ ░░▒░▒ ▒▒   ▓▒█░ ▒▒▓  ▒    ██▒▒▒     ▒▓▒▒░    ▒▒   ▓▒█░░ ▒░   ▒ ▒ ░░ ▒░ ░    //
+//      ░ ░▒  ░ ░ ▒ ░▒░ ░  ▒   ▒▒ ░ ░ ▒  ▒  ▓██ ░▒░     ▒ ░▒░     ▒   ▒▒ ░░ ░░   ░ ▒░ ░ ░  ░    //
+//      ░  ░  ░   ░  ░░ ░  ░   ▒    ░ ░  ░  ▒ ▒ ░░      ░ ░ ░     ░   ▒      ░   ░ ░    ░       //
+//            ░   ░  ░  ░      ░  ░   ░     ░ ░         ░   ░         ░  ░         ░    ░  ░    //
+//                                  ░       ░ ░                                                 //
+//////////////////////////////////////////////////////////////////////////////////////////////////
+//Staff assistant who set up a store in a maintenance tunnel somewhere, trading stolen shit.
+/obj/npc/trader/blackmarket/shady
+	icon = 'icons/obj/trader.dmi'
+	icon_state = "shady"
+	picture = "shady.gif"
+	name = "Shady Jane's Back-Alley"
+	trader_area = "/area/diner/blackmarket"
+	angrynope = "*You can't establish a connection with Shady Jane*"
+	whotext = "None of your business, nerd."
+	New()
+		..()
+		greeting= {"Got some goods for sale, and creds if you're looking to pawn some stuff off. No questions asked."}
+		portrait_setup = "<img src='[resource("images/traders/[src.picture]")]'><HR><B>[src.name]</B><HR>"
+		sell_dialogue = "Here's what I want."
+		buy_dialogue = "Feast your eyes:"
+		successful_purchase_dialogue = list("Yep, let's trade.",
+			"Adding it to your cart.",
+			"You got it.",
+			"Lots of things you can do with those!")
+		failed_sale_dialogue = list("I don't really want that.",
+			"Nah.",
+			"Not quite what I'm looking for.",
+			"Don't want it.")
+		successful_sale_dialogue = list("Thanks, pal.",
+			"Deal!",
+			"Alright, I'll take it.")
+		failed_purchase_dialogue = list("Can't afford it.",
+			"Not enough cash, pal.",
+			"You sure you can afford that?")
+		pickupdialogue = "Enjoy!"
+		pickupdialoguefailure = "Nothing in your cart, pal."
+//////////////////////////////////////////////////////////////
+///███████ ███████ ██      ██      ██ ███    ██  ██████     //
+///██      ██      ██      ██      ██ ████   ██ ██       ██ //
+///███████ █████   ██      ██      ██ ██ ██  ██ ██   ███    //
+///     ██ ██      ██      ██      ██ ██  ██ ██ ██    ██ ██ //
+///███████ ███████ ███████ ███████ ██ ██   ████  ██████     //
+//////////////////////////////////////////////////////////////
+		src.goods_sell += new /datum/commodity/blackmarket/shady/cursedmask(src)
+		src.goods_sell += new /datum/commodity/blackmarket/shady/omnitool(src)
```

Comment:
Probably should check to see if someone is a syndie traitor, for all the gear that's syndicate in origin. Randomly having to adapt to some nerd baton stunning you down and then cluwneing you is not gonna be fun.

Reply:
it's an acid mask not a cluwne mask